addons/website/static/src/scss/website.scss

```diff
@@ -1422,24 +1422,38 @@ $-o-carousel-controllers-size: var(--CarouselControllers-size, #{map-get($spacer
 
 // Parallax
 .parallax {
-    // TODO this introduces a limitation: no dropdown will be able to
-    // overflow. Maybe there is a better way to find.
-    &:not(.s_parallax_no_overflow_hidden) {
+    // Kept for compatibility. Before "s_parallax_bg" was wrapped by an
+    // intermediate parent element to clip it ("s_parallax_bg_wrap"), its direct
+    // parent was the section with the "parallax" class, which had the CSS rule
+    // "overflow: hidden". That was not the right solution because it introduced
+    // a limitation: no dropdown could overflow, for example.
+    &:not(.s_parallax_no_overflow_hidden):not(:has(> .s_parallax_bg_wrap)) {
         overflow: hidden;
     }
 
-    > .s_parallax_bg {
+    > .s_parallax_bg, > .s_parallax_bg_wrap > .s_parallax_bg {
         @extend %o-we-background-layer;
+
+        &.o_bg_img_opt_repeat {
+            background-repeat: repeat;
+        }
```

Comment:
What is that one? Probably deserve a comment and probably its own commit?

Reply:
> What is that one? Probably deserve a comment and probably its own commit?

Adding the class `.s_parallax_bg_wrap` in the selector increased the priority of the rule
`background-repeat: no-repeat;` which is inside `%o-we-background-layer;` I add a comment 👍🏼